src/models/NetworkAppController.js

```diff
@@ -267,4 +265,8 @@ export default class NetworkAppController {
   isContractDeployed(contractAlias) {
     return !this.isApplicationContract(contractAlias) || !_.isEmpty(this.networkPackage.contracts[contractAlias]);
   }
-}
\ No newline at end of file
+
+  areSameStdlib(aStdlib, anotherStdlib) {
+    return aStdlib.name === anotherStdlib.name && aStdlib.version === anotherStdlib.version
+  }
```

Comment:
-1 to sneaking unrelated refactors in PRs. It's just a detail though, let's leave this one for now, but just keep it in mind for future changes.